rviz_default_plugins/src/rviz_default_plugins/displays/camera/camera_display.cpp

```diff
@@ -620,6 +621,28 @@ Ogre::Matrix4 CameraDisplay::calculateProjectionMatrix(
   return proj_matrix;
 }
 
+static Ogre::Vector4 calculateScreenCorners(
+  const sensor_msgs::msg::CameraInfo::ConstSharedPtr info,
+  const Ogre::Vector2 & zoom)
```

Comment:
There's a [clang warning](https://build.ros2.org/job/Rpr__rviz__ubuntu_jammy_amd64/86/clang-tidy/new/) about this not being used. I suspect moving the definition above the place where it is used above will resolve it.

Reply:
Same issue is causing a gcc error: https://build.ros2.org/job/Rpr__rviz__ubuntu_jammy_amd64/86/gcc/new/#issuesContent

Reply:
Thank you very much. You're absolutely right. Sorry for this mistake.

Reply:
I force-pushed an update